spec/jobs/scraper/apprenticeship_bulletins_job_spec.rb

```diff
@@ -0,0 +1,53 @@
+require "rails_helper"
+
+RSpec.describe Scraper::AppreticeshipBulletinsJob, type: :job do
+  describe "#perform" do
+    context "when files have not been downloaded previously" do
+      it "downloads docx files to a standards import record" do
+        stub_responses
+        expect {
+          described_class.new.perform
+        }.to change(StandardsImport, :count).by(3)
+
+        standard_import = StandardsImport.last
+        expect(standard_import.files.count).to eq 1
+        expect(standard_import.name).to eq "https://www.apprenticeship.gov/sites/default/files/bulletins/Bulletin%202023-50%20New%20NOF%20Appren%20Occ%20Jr%20Cloud%20Engineer.docx"
+        expect(standard_import.organization).to eq "2023-50"
+        expect(standard_import.notes).to eq "From Scraper::AppreticeshipBulletinsJob #{Scraper::AppreticeshipBulletinsJob::BULLETIN_LIST_URL}"
+      end
+    end
```

Comment:
Could you also add a test that checks the "Attachments. None" skip?

Reply:
@littleforest added